test/v2/integration/trading.ts

```diff
@@ -0,0 +1,45 @@
+import { coroutine as co } from 'bluebird';
+import should = require('should');
+import * as nock from 'nock';
+
+const TestV2BitGo = require('../../lib/test_bitgo');
+
+describe('V2 Trading', function() {
+  let bitgo;
+  let basecoin;
+  let wallets;
+  let wallet;
+  let trading;
+
+  before(co(function *() {
+    nock.restore();
+
+    bitgo = new TestV2BitGo({ env: 'dev', microservicesUri: 'https://app.bitgo-dev.com' });
```

Comment:
This is now disclosing the microservices uri, are we comfortable with that? (that ship has now sailed however, since this PR is public).